pkg/engine/cas_template_engine.go

```diff
@@ -261,20 +260,36 @@ func (c *CASEngine) Run() (output []byte, err error) {
 
 // create creates a CAS entity
 func (c *CASEngine) Create() (output []byte, err error) {
+	err = c.AddConfigToConfigTLP(c.casTemplate.Spec.Defaults)
```

Comment:
How about pushing these into Run() method.... It will help us avoid redundancy...